# from scripts.scraper1 import main as _1         
# from scripts.scraper2 import main as _2       
# from scripts.scraper3 import main as _3       
# from scripts.scraper4 import main as _4       
# from scripts.scraper5 import main as _5       
# from scripts.scraper6 import main as _6       
# from scripts.scraper7 import main as _7       
# from scripts.scraper8 import main as _8       
# from scripts.scraper9 import main as _9       
# from scripts.scraper10 import main as _10       
# from scripts.scraper11 import main as _11       
# from scripts.scraper12 import main as _12       
# from scripts.scraper13 import main as _13       
# from scripts.scraper14 import main as _14           # Not completed! (job is not exist)     
# from scripts.scraper15 import main as _15       
# from scripts.scraper16 import main as _16       
# from scripts.scraper17 import main as _17       
# from scripts.scraper18 import main as _18       
# from scripts.scraper19 import main as _19       
from scripts.scraper20 import main as _20       


def run():
    # try: _1() 
    # except Exception as e: print(f'Scraper1: {e}')
    
    # try: _2() 
    # except Exception as e: print(f'Scraper2: {e}')
    
    # try: _3() 
    # except Exception as e: print(f'Scraper3: {e}')
    
    # try: _4() 
    # except Exception as e: print(f'Scraper4: {e}')
    
    # try: _5() 
    # except Exception as e: print(f'Scraper5: {e}')
    
    # try: _6() 
    # except Exception as e: print(f'Scraper6: {e}')
    
    # try: _7() 
    # except Exception as e: print(f'Scraper7: {e}')
    
    # try: _8() 
    # except Exception as e: print(f'Scraper8: {e}')
    
    # try: _9() 
    # except Exception as e: print(f'Scraper9: {e}')
    
    # try: _10() 
    # except Exception as e: print(f'Scraper10: {e}')
    
    # try: _11() 
    # except Exception as e: print(f'Scraper11: {e}')
    
    # try: _12() 
    # except Exception as e: print(f'Scraper12: {e}')
    
    # try: _13() 
    # except Exception as e: print(f'Scraper13: {e}')
    
    # try: _14() 
    # except Exception as e: print(f'Scraper14: {e}')
    
    # try: _15() 
    # except Exception as e: print(f'Scraper15: {e}')
    
    # try: _16() 
    # except Exception as e: print(f'Scraper16: {e}')
    
    # try: _18() 
    # except Exception as e: print(f'Scraper18: {e}')
    
    # try: _19() 
    # except Exception as e: print(f'Scraper19: {e}')
    
    try: _20() 
    except Exception as e: print(f'Scraper20: {e}')
    
    
if __name__ == "__main__":
    run()